feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/send/TransferValidityChecks.kt

```diff
@@ -55,11 +55,13 @@ private fun BaseFragment<*>.showTransferError(
     status: TransferValidityLevel.Error.Status,
     errorConfirmed: DialogClickHandler?
 ) {
-    val messageRes = when (status) {
-        TransferValidityLevel.Error.Status.NotEnoughFunds -> R.string.choose_amount_error_too_big
+    val (titleRes, messageRes) = when (status) {
+        TransferValidityLevel.Error.Status.NotEnoughFunds -> R.string.common_error_general_title to R.string.choose_amount_error_too_big
+        TransferValidityLevel.Error.Status.DeadRecipient -> R.string.wallet_send_dead_recipient_title to R.string.wallet_send_dead_recipient_message
```

Comment:
rip

Reply:
good thing that they can be revived by relevantly small price, truly the future we all want